backend/dal/Pims.Dal.csproj

```diff
@@ -40,6 +40,12 @@
     <Content Include="Migrations\**\*.sql">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Update="Migrations\v01.07.00\Up\PostUp\01-ViewProperties.sql">
```

Comment:
I was under the impression the above covered this, so this is redundant @Fosol

Reply:
The above does cover this.  You don't need to add these new `<Content ...>` nodes.